tests/sentry/grouping/snapshots/test_variants/test_event_hash_variant/newstyle@2023_01_11/callee_guaranteed.pysnap

```diff
@@ -141,25 +141,25 @@ system:
           frame (ignored by stack trace rule (category:threadbase -group v-group))
             function*
               "UIApplicationMain"
```

Comment:
This frame and the ones above are discarded for grouping because of this rule:
https://github.com/getsentry/sentry/blob/85781853c026931f81b376f908e4c8b37ebf43c0/src/sentry/grouping/enhancer/enhancement-configs/newstyle%402023-01-11.txt#L158